sh_scripts/templates/new-realm.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
@nvunnamm Are you sure this change is correct? It looks like somebody is hardcoding this stuff for some reason. I don't remember us changing this when we were pairing. Please analyze and comment.

Reply:
Yes Conrad I too don't remember us changing I will revert it back.

Reply:
I'm not sure if that file even gets used or if it is intended to be copied and filled out. Can you see if it is referenced by anything?